frontend/src/utils/utils.ts

```diff
@@ -0,0 +1,27 @@
+export const classSet = (input?: any): string => {
```

Comment:
Can you add some documentation `jsdoc` to these?

Reply:
tech-debt yes